src/main/webapp/course.html

```diff
@@ -0,0 +1,145 @@
+    <script
+      type="text/javascript"
+      src="https://www.gstatic.com/charts/loader.js"
+    ></script>
+    <script src="courseLoader.js"></script>
+    <link rel="stylesheet" href="course-style.css"/>
+  </head>
+  <body onload="fillTitles()">
+    <nav class="navbar navbar-dark bg-dark">
+      <span class="navbar-brand mb-0 h1">App Name</span>
+    </nav>
+
+    <div class="jumbotron jumbotron-fluid my-0">
+      <div class="container">
+        <h1 class="display-4 text-center" id = "course-name">Course Name</h1>
+        <p class="lead text-center" id="term-name">Term Name</p>
+      </div>
+    </div>
+
+    <div class="flex-container">
+      <div class="flex-child" id="hours-chart" ></div>
+      <div class="flex-child" id="diff-chart"></div>
+    </div>
+
+    <div class="flex-container">
+      <div class="flex-child" id="prof-chart" ></div>
+      <div class="flex-child" id="term-chart"></div>
+    </div>
+
+    <div class="my-3 p-3 bg-white course-comments">
+        <h1 class="border-bottom border-gray p-5 mb-0 text-center">Course Comments</h1>
+        <div class="media text-muted pt-3">
+        <p class="media-body pb-3 mb-0 small lh-125 border-bottom border-gray">
+            <strong class="d-block text-gray-dark">@username</strong>
+            Donec id elit non mi porta gravida at eget metus. Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus.
+        </p>
+        </div>
+        <div class="media text-muted pt-3">
+        <p class="media-body pb-3 mb-0 small lh-125 border-bottom border-gray">
+            <strong class="d-block text-gray-dark">@username</strong>
+            Donec id elit non mi porta gravida at eget metus. Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus.
+        </p>
+        </div>
+    </div>
+
+    <div class="my-3 p-3 bg-white prof-comments">
+        <h1 class="border-bottom border-gray p-5 mb-0 text-center">Professor Comments</h1>
+        <div class="media text-muted pt-3">
+        <p class="media-body pb-3 mb-0 small lh-125 border-bottom border-gray">
+            <strong class="d-block text-gray-dark">@username</strong>
+            Donec id elit non mi porta gravida at eget metus. Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus.
+        </p>
+        </div>
+        <div class="media text-muted pt-3">
+        <p class="media-body pb-3 mb-0 small lh-125 border-bottom border-gray">
+            <strong class="d-block text-gray-dark">@username</strong>
+            Donec id elit non mi porta gravida at eget metus. Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus.
```

Comment:
Wait out of curiosity what does it say here? hahaha

Reply:
No clue, I think its randomly generate text from an example I used

Reply:
According to Google Translate, it doesn't actually mean anything in English - just gibberish lol

Reply:
rip sad :(